### PR TITLE
Add mask + formatting to Database.Grid

### PR DESCRIPTION
**Pull request by GordonSmith**
Allow Class.mixin to augment class list.
Remove existing Table formatting and replace with fields.
Allow persist suuport for non Widget classes which mixin PropertyExt.
Comms timeout previously set to 1 (for testing?)

Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@dtsnell4 @mlzummo Please Review
rawgit.com/GordonSmith/Visualization/DB_FORMATTING/demos/dermatology.html?src/other/Table.formatted

File: src/common/Widget.js [hunk 1]
```diff
@@ -127,13 +129,27 @@
         this._id = _;
         return this;
     };
-
+/*
```
Comment (mlzummo) [inline src/common/Widget.js]: commented code?

**Comment by GordonSmith** [comment]
@dtsnell4 @mlzummo Please Review (fixed tests and removed comments / console logging)

**Comment by dtsnell4** [comment]
Looks ok to me.

**Comment by dtsnell4** [comment]
Actually, the sorting no longer works in the table, but that should be a new issue?

**Comment by GordonSmith** [comment]
@dtsnell4 New PR with sort fixed - take note that the date column sorts correctly even though it has a poor choice of format string!

**Comment by dtsnell4** [comment]
It is sorting now, but doesn't seem to be quite correct.  Latitude and String columns sort correctly, but the two Longitude columns do not.  
capture

**Comment by dtsnell4** [comment]
Of course if I switch the type from string to number they're fine ;-)

**Comment by GordonSmith** [comment]
This highlights an issue - I needs a new "type" of "native" of "default" or "" which leaves the underlying type alone...  rather than defaulting to "string"...

**Comment by GordonSmith** [comment]
@dtsnell4 @mlzummo Please rereview

**Comment by dtsnell4** [comment]
Looks good to me

**Comment by mlzummo** [comment]
looks good

**Pull request merged**